### PR TITLE
avro-c avro-cpp avro-tools 1.11.3

### PR DESCRIPTION
**Pull request by iMichka**
Releases on https://avro.apache.org/ are gone.
Tar.gz file on Github is broken so falling back to zip file

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by iMichka** [comment]
Can't use zip files? I wanted to, because using tar gives the following error:

```
/usr/bin/env tar --extract --no-same-owner --file /Users/michkapopoff/Library/Caches/Homebrew/downloads/98ef0090c777e10b603b4023cee466e218ddfa016a761181221103cfd8ae28bd--avro-release-1.11.3.tar.gz --directory /private/tmp/d20230927-6350-12x8mgy
avro-release-1.11.3/lang/java/mapred/src/test/resources/org/apache/avro/mapreduce/mapreduce-test-input.avro/.part-r-00000.avro.crc: Failed to restore metadata
tar: Error exit delayed from previous errors.
Error: Failure while executing; `/usr/bin/env tar --extract --no-same-owner --file /Users/michkapopoff/Library/Caches/Homebrew/downloads/98ef0090c777e10b603b4023cee466e218ddfa016a761181221103cfd8ae28bd--avro-release-1.11.3.tar.gz --directory /private/tmp/d20230927-6350-12x8mgy` exited with 1. Here's the output:
avro-release-1.11.3/lang/java/mapred/src/test/resources/org/apache/avro/mapreduce/mapreduce-test-input.avro/.part-r-00000.avro.crc: Failed to restore metadata
tar: Error exit delayed from previous errors.
```

**Comment by iMichka** [comment]
The file is there since years ... https://github.com/apache/avro/tree/master/lang/java/mapred/src/test/resources/org/apache/avro/mapreduce/mapreduce-test-input.avro

Maybe it's an issue now because I'm using the tar files from Github ...

**Review by ZhongRuoyu — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

File: Formula/a/avro-cpp.rb [hunk 1]
```diff
@@ -1,9 +1,12 @@
 class AvroCpp < Formula
   desc "Data serialization system"
   homepage "https://avro.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=avro/avro-1.11.2/cpp/avro-cpp-1.11.2.tar.gz"
-  mirror "https://archive.apache.org/dist/avro/avro-1.11.2/cpp/avro-cpp-1.11.2.tar.gz"
```
Comment (chenrui333) [inline Formula/a/avro-cpp.rb:5]: image

File: Formula/a/avro-c.rb [hunk 2]
```diff
@@ -1,9 +1,12 @@
 class AvroC < Formula
   desc "Data serialization system"
   homepage "https://avro.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=avro/avro-1.11.2/c/avro-c-1.11.2.tar.gz"
-  mirror "https://archive.apache.org/dist/avro/avro-1.11.2/c/avro-c-1.11.2.tar.gz"
```
[the remark this answers is not in the document]
Comment (iMichka) [reply Formula/a/avro-c.rb:5]: Check my comment in the code, all these tar files can not be opened on macOS ...

**Pull request merged**